src/gui/guiFormSpecMenu.cpp

```diff
@@ -2687,6 +2687,26 @@ bool GUIFormSpecMenu::parseStyle(parserData *data, const std::string &element, b
 	return true;
 }
 
+void GUIFormSpecMenu::parseSetFocus(const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if (parts.size() <= 2 ||
+		(parts.size() > 2 && m_formspec_version > FORMSPEC_API_VERSION))
+	{
+		if (!m_is_form_regenerated && // Never focus on resizing
```

Comment:
How about early returns here? The code will be longer, but much easier to understand.

```C++
if (m_is_form_regenerated)
	return; // Never focus on resizing
```